packages/react/runtime/src/lynx/performance.ts

```diff
@@ -22,8 +22,19 @@ enum PerformanceTimingKeys {
   patch_changes_end,
   hydrate_parse_snapshot_start,
   hydrate_parse_snapshot_end,
+  mtsRenderStart,
```

Comment:
Can we use the same naming convention?
```suggestion
  mts_render_start,
```

Reply:
There is a topic here. The SDK Performance API uses camel case style, such as "[PipelineEntry](https://lynxjs.org/zh/api/lynx-api/performance-api/performance-entry/pipeline-entry.html)." Should we modify the timing keys in the framework to camel case as well, and let the SDK ensure compatibility with the old interfaces? (However, this will cause an issue where the new version of the framework combined with the old version of the SDK will lead to changes in the parameter style received by the client.)